CONTRIBUTING.md

```diff
@@ -289,6 +299,32 @@ do not decrease overall code coverage of the codebase - this is aligned with our
 goal to increase coverage over time. Keep track of execution time for your unit
 tests and try to keep them as short as possible.
 
+#### Library Recommendations 
+
+To keep testing practices consistent across the project, it is advised to use this libraries under
+these circumstances:
+
+- For assertions to validate expecations, use `"github.com/stretchr/testify/assert"`
+- For assertions that are required to continue the test, use `"github.com/stretchr/testify/require"`
+- For mocking external resources, use `"github.com/stretchr/testify/mock"`
+- For validating HTTP traffic interactions, `"net/http/httptest"`
+- For generating test data, use `???`
+
+### Integration Testing
+
+Integration testing is encourage throughout the project, docker images can be used in order to facility
+a local version. The recommend library to control docker images throughout testing is `???`.
```

Comment:
I believe I have seem packages use a docker controller to manage which would be good for integration tests.